src/panels/lovelace/cards/hui-light-card.ts

```diff
@@ -0,0 +1,312 @@
+import { html, LitElement, PropertyValues } from "@polymer/lit-element";
+import { fireEvent } from "../../../common/dom/fire_event.js";
+import { styleMap } from "lit-html/directives/styleMap.js";
+import computeStateName from "../../../common/entity/compute_state_name.js";
+import stateIcon from "../../../common/entity/state_icon.js";
+import { jQuery } from "../../../resources/jquery";
+
+import "../../../components/ha-card.js";
+import "../../../components/ha-icon.js";
+import { roundSliderStyle } from "../../../resources/jquery.roundslider";
+
+import { HomeAssistant, LightEntity } from "../../../types.js";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceCard, LovelaceConfig } from "../types.js";
+import { longPress } from "../common/directives/long-press-directive";
+
+const lightConfig = {
+  radius: 80,
+  step: 1,
+  circleShape: "pie",
+  startAngle: 315,
+  width: 5,
+  min: 1,
+  max: 100,
+  sliderType: "min-range",
+  lineCap: "round",
+  handleSize: "+12",
+  showTooltip: false,
+};
+
+interface Config extends LovelaceConfig {
+  entity: string;
+  name?: string;
+}
+
+export class HuiLightCard extends hassLocalizeLitMixin(LitElement)
+  implements LovelaceCard {
+  public hass?: HomeAssistant;
+  private _config?: Config;
+  private _brightnessTimout?: NodeJS.Timer;
```

Comment:
We're running in a browser, not under NodeJS. Also:

![image](https://user-images.githubusercontent.com/1444314/47678936-3dfe9400-dbc3-11e8-82bd-2a1f0f75880a.png)

`number`

Reply:
When changing to `number` I am getting `Type Time is not assignable to Type Number` here: `this._brightnessTimout = setTimeout(() => {`

I think Node is messing with things or something.. Because researching setTimeout shows it returns a "unique" number that is used when clearing the timeout

Reply:
Fix this by using window.setTimeout so node doesnt overload